### DISCUSSION TITLE
icon in table cells?

### DISCUSSION THREAD

**Discussion by magickaito** [Budibase/budibase#5998] [Help] [answered]
![Screenshot 2022-05-20 at 4 06 06 PM](https://user-images.githubusercontent.com/1195573/169483516-92be284f-02cd-4468-8aa0-492ab3edd2c8.jpg)

In the Budibase app listing UI, there is this elegant small green dot beside the App status. How to achieve this?

My closest hack is to put an unicode emoji right into the list of options, but it looks kind of ugly:

![Screenshot 2022-05-20 at 4 09 33 PM](https://user-images.githubusercontent.com/1195573/169484034-fa69ba3a-88da-47fe-86a9-cc3d96b36f41.jpg)

Also I think it may have other unforeseen problem if the database has emoji. 

Any other methods to achieve that look?

**Comment by melohagan** [comment, marked as answer]
Hi @magickaito 

Currently you can't get exactly what you want: 
- https://github.com/Budibase/budibase/discussions/4889
- https://github.com/Budibase/budibase/issues/3273

In the meantime, what you could do is add a container with image elements to the row and play around with the formatting. 
![Screenshot 2022-05-20 at 09 38 59](https://user-images.githubusercontent.com/101575380/169489404-229d2f44-e009-42fe-b8d6-5e908f2212f5.png)

You need to add conditional UI (https://docs.budibase.com/docs/conditions) to display the correct dot, e.g.
![Screenshot 2022-05-20 at 09 40 13](https://user-images.githubusercontent.com/101575380/169489621-d9949f41-3e5f-4246-8162-1154e587df62.png)

I also think the green dot you have at the moment looks decent. It also looks different on other platforms: https://emojiguide.com/symbols/green-circle/

**Reply by magickaito** [reply]
got it thanks!
